app/src/main/java/com/plaid/linksample/MainActivity.kt

```diff
@@ -66,8 +67,8 @@ class MainActivity : AppCompatActivity() {
     setContentView(R.layout.activity_main)
     result = findViewById(R.id.result)
 
-    val fab = findViewById<FloatingActionButton>(R.id.open_link)
-    fab.setOnClickListener {
+    val button = findViewById<View>(R.id.open_link)
```

Comment:
Generally we should prefer the highest level interface (so that switching out implementations doesn't cause issues like it did here), and I think from a reader perspective, a clickable View vs Button isn't a super important distinction